src/discussions/posts/post/Post.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import postShape from './post-shape';
+import PostIcons from '../post-icons/PostIcons';
+
+function Post({ post }) {
+  return (
+    <div className="discussion-post d-flex flex-column">
```

Comment:
This is the threads UI, and it doesn't look close to the one in the mockups.
There's borders around the components, the sizes look different and the post editor doesn't show up.

The buttons on the post UI don't work.

**Mockup:**
![image](https://user-images.githubusercontent.com/27893385/91783500-d5293e00-ebd6-11ea-99d7-7a31387351d9.png)

**Implementation:**
![image](https://user-images.githubusercontent.com/27893385/91783515-dce8e280-ebd6-11ea-8de0-e3f1ff493138.png)

Reply:
@giovannicimolin This is mostly about the UI, it's OK if the buttons don't work yet, and the post editor work is separate.

Reply:
@giovannicimolin @xitij2000

The issue with the mockups is that they do not indicate how a post with replies should look. What does the full thread look like when there are more than the original post.  If the design I've implemented using the `Card` UI that was used for the Threads component is not what we want, I am happy to rework this. The `Card` UI was the simplest way to delineate the differences between each individual post/comment/reply.

I'll look into adding the post editor. Do we have a component that we use for this? I assume there is a specific HTML/JSX/React snippet I should be using for this and that I don't need to reimplement all of it.

As for the buttons, I was under the impression that the extent of the functionality I was to implement for this was the interactions that send a user to the existing components (Topic, Thread, Post, Comment etc) and that any additional functionality would be added to the MFE separately.

Reply:
> The issue with the mockups is that they do not indicate how a post with replies should look.

You can implement it in a straight line for now. 

> I'll look into adding the post editor. Do we have a component that we use for this? I assume there is a specific HTML/JSX/React snippet I should be using for this and that I don't need to reimplement all of it.

Do not do that. The scope of this ticket is limited to the things I mentioned on the ticket. This ticket is already going way beyond that scope. 

> As for the buttons, I was under the impression that the extent of the functionality I was to implement for this was the interactions that send a user to the existing components (Topic, Thread, Post, Comment etc) and that any additional functionality would be added to the MFE separately.

Correct, but it's useful to add stub code to allow exploring all the UI.

Reply:
@xitij2000

> > The issue with the mockups is that they do not indicate how a post with replies should look.
>
> You can implement it in a straight line for now.

So, to confirm, the desired result is to remove the `Card` UI, and then simply render all comments in a straight line one after another with no separator? Also, how should the Replies be rendered since the original edx-platform implementation for discussions shows them as an inline thread immediately after the comment they are replying to?

Reply:
The desired result is to get the UI to look as close to the mockup as possible. If using Cards helps, then you use them. If not, you don't. In this case they seem to do more harm than good. 

This is a first-pass implementation, and the idea is to get a very close representation of what a single comment/post looks like since we have that mockup.